Marlin/LED_Strip.cpp

```diff
@@ -0,0 +1,32 @@
+// RGB Led Strip Control
+ 
+#include "Marlin.h"
+#include "temperature.h" 
+#include "Configuration.h"
+#include "ultralcd.h"
+#include "LED_Strip.h"
+
+#if ENABLED(LED_STRIP)
+
+// Set color by RGB values
+void rgb_to_led_strip(int red, int green, int blue) {    
+  analogWrite(REDPIN, red);
+  analogWrite(GREENPIN, green);
+  analogWrite(BLUEPIN, blue);
```

Comment:
It's not completely settled yet, but we use the technique to check in `pinsDebug.h`. For now we can actually the old "trick" that's been used up to now:
```cpp
digitalWrite(RGB_LED_RED_PIN, r ? HIGH : LOW);
analogWrite(RGB_LED_RED_PIN, r);
```
If the pin isn't analog (hence PWM) then the second command just doesn't do anything. It's already implemented this way in the existing `M150`.